src/main/java/mate/academy/internetshop/model/Role.java

```diff
@@ -6,6 +6,11 @@
 
     public Role(RoleName roleName) {
         this.roleName = roleName;
+        if (roleName.equals(RoleName.ADMIN)) {
+            this.id = 1L;
+        } else {
+            this.id = 2L;
+        }
```

Comment:
that's were)) imagine we will have 10 roles or even more?

Reply:
Should i implement role service?
It seems the solution, but big one...

Reply:
Decided to use name of Role to get its Id. Maybe, not brilliant, but alternative is to implement role service :)